Fix trace_agent_ctl deprecated flag in use (#209)
DataDog/datadog-agent-boshrelease: CHANGELOG.md

# Changelog

## 4.16.0 / 2023-11-20

> * [Fixed] Fix trace_agent_ctl fails to start. Deprecated pid flag being used. Updated to --pidfile. See [#209](https://github.com/DataDog/datadog-agent-boshrelease/pull/209)

Reviewer: ```suggestion
* [Fixed] Fix `trace_agent_ctl` failing to start. Deprecated the `pid` flag being used, and upgraded to `--pidfile`. See [#209](https://github.com/DataDog/datadog-agent-boshrelease/pull/209).
```
